tv-shows/src/components/features/review/ReviewList.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from "@testing-library/react";
+import { ReviewList } from "./ReviewList";
+import { ReviewItem } from "./ReviewItem";
+import { IReviewList } from "@/typings/Review.type";
+
+jest.mock("./ReviewItem", () => ({
+  ReviewItem: jest.fn(({ review }) => (
+    <div data-testid="mock-review-item">
+      <p>{review.comment}</p>
+    </div>
+  )),
```

Comment:
Rekli smo da nas u listi ne zanima kako review item izgleda, već nas zanima da se "pozvao/renderao" točan broj puta sa točnim propsima.
Unutar review item-a ćemo testirati njegov content.
Baci oko na kod s predavanja kako sam mockao todo list item.